lib/util/clone.ts

```diff
@@ -1,11 +1,10 @@
-import { quickStringify } from './stringify';
+import { klona } from 'klona/json';
 
 /**
  * Creates a deep clone of an object.
  * @deprecated Use {@link structuredClone} instead.
  * @param input The object to clone.
  */
-export function clone<T>(input: T | null = null): T {
-  const stringifiedInput = quickStringify(input);
-  return stringifiedInput ? JSON.parse(stringifiedInput) : null;
+export function clone<T = unknown>(input: T): T {
+  return klona(input);
```

Comment:
what's the benefit to use klona? if I remember correctly it has problems with circular references.

Reply:
Yes, but I'm not sure we have them, nor we need to have it by design. So, let's just treat them as bugs when `RangeError` has encountered.

Reply:
In exchange, we'll receive speed for this particular operation

Reply:
https://github.com/lukeed/klona/issues/40

Reply:
https://github.com/lukeed/klona#klonalite

we probably need the lite variant to support date and regex

Reply:
Fixed